src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/tests/test_resource_list_odata_filter.py

```diff
@@ -4,9 +4,45 @@
 # --------------------------------------------------------------------------------------------
```

Comment:
Please note that the test files have been moved. You need to rebase on the last master and apply the changes to the test files are new location.